worlds/civ_6/Regions.py

```diff
@@ -0,0 +1,155 @@
+    menu = Region("Menu", player, world.multiworld)
+    world.multiworld.regions.append(menu)
+
+    has_progressive_eras = options.progression_style == "eras_and_districts"
+    has_goody_huts = options.shuffle_goody_hut_rewards
+    has_boosts = options.boostsanity
+
+    regions: List[Region] = []
+    for era in EraType:
+        era_region = Region(era.value, player, world.multiworld)
+        era_locations: Dict[str, Optional[int]] = {location.name: location.code for _key,
+                                                   location in world.location_by_era[era.value].items()}
+
+        if not has_progressive_eras:
+            era_locations = {key: value for key, value in era_locations.items() if key.split("_")[0] != "ERA"}
+        if not has_goody_huts:
+            era_locations = {key: value for key, value in era_locations.items() if key.split("_")[0] != "GOODY"}
+        if not has_boosts:
+            era_locations = {key: value for key, value in era_locations.items() if key.split("_")[0] != "BOOST"}
+
+        era_region.add_locations(era_locations, CivVILocation)
+
+        regions.append(era_region)
+        world.multiworld.regions.append(era_region)
+
+    menu.connect(world.get_region(EraType.ERA_ANCIENT.value))
+
+    world.get_region(EraType.ERA_ANCIENT.value).connect(
+        world.get_region(EraType.ERA_CLASSICAL.value), None,
+        lambda state: has_required_items(
+            state, EraType.ERA_ANCIENT, world)
+    )
+
+    world.get_region(EraType.ERA_CLASSICAL.value).connect(
+        world.get_region(EraType.ERA_MEDIEVAL.value), None, lambda state: has_required_items(
+            state, EraType.ERA_CLASSICAL, world)
+    )
+
+    world.get_region(EraType.ERA_MEDIEVAL.value).connect(
+        world.get_region(EraType.ERA_RENAISSANCE.value), None, lambda state: has_required_items(
+            state, EraType.ERA_MEDIEVAL, world)
+    )
+
+    world.get_region(EraType.ERA_RENAISSANCE.value).connect(
+        world.get_region(EraType.ERA_INDUSTRIAL.value), None, lambda state: has_required_items(
+            state, EraType.ERA_RENAISSANCE, world)
+    )
+
+    world.get_region(EraType.ERA_INDUSTRIAL.value).connect(
+        world.get_region(EraType.ERA_MODERN.value), None, lambda state: has_required_items(
+            state, EraType.ERA_INDUSTRIAL, world)
+    )
+
+    world.get_region(EraType.ERA_MODERN.value).connect(
+        world.get_region(EraType.ERA_ATOMIC.value), None, lambda state: has_required_items(
+            state, EraType.ERA_MODERN, world)
+    )
+
+    world.get_region(EraType.ERA_ATOMIC.value).connect(
+        world.get_region(EraType.ERA_INFORMATION.value), None, lambda state: has_required_items(
+            state, EraType.ERA_ATOMIC, world)
+    )
+
+    world.get_region(EraType.ERA_INFORMATION.value).connect(
+        world.get_region(EraType.ERA_FUTURE.value), None, lambda state: has_required_items(state, EraType.ERA_INFORMATION, world))
```

Comment:
The formatting is different for this one
```suggestion
    world.get_region(EraType.ERA_INFORMATION.value).connect(
        world.get_region(EraType.ERA_FUTURE.value), None, lambda state: has_required_items(
            state, EraType.ERA_INFORMATION, world)
    )
```